tests/Unit/BasicSyntaxTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Mockery as m;
+use RicorocksDigitalAgency\Soap\Request\Request;
+use RicorocksDigitalAgency\Soap\Response\Response;
+
+it('can obtain a WSDL', function () {
+    $mock = m::mock(Request::class);
+    $mock->shouldReceive('beforeRequesting', 'afterRequesting', 'to')->andReturnSelf()
+        ->shouldReceive('functions')->andReturn(
+            [
+                'AddResponse Add(Add $parameters)',
```

Comment:
We've set up the mock return value but then not asserted against it. Could we do that? And with it being a mock, could we reduce the number of items in the array?

Reply:
Updated